glitchdotcom/Glitch-Community
# Sitelinks searchbox support

## Links
* https://search-seo.glitch.me 
* https://app.clubhouse.io/glitch/story/7718/add-markup-to-enable-searching-glitch-com-from-the-search-result-for-glitch
* [Google Search Rich Results test results](https://search.google.com/test/rich-results?id=24VCMl3snCH-ufYK3UyOag) - kept erroring on the full page, so this tests the raw HTML instead. 

## GIF/Screenshots:
![results from Search Rich Results test results link above](https://user-images.githubusercontent.com/4480480/73309009-0a0b8780-41e7-11ea-9319-4138cb39c048.png)


## Changes:
* adds some ld-json that tells Google/other search engines how to search glitch.com

## How To Test:
* Make sure the page still looks fine
* Can run the code or page through https://search.google.com/test/rich-results 
* Take the url in `ldJson.potentialAction.target` (line 397), replace `{search_term_string}` with other characters, and you should arrive at the glitch.com search results page with results for those characters.
* *After deploy*, Cassey should  check the Google search console and confirm the configuration works in production, too.

## Feedback I'm looking for:
whatever
